ts-client/arkeo.arkeo/index.ts

```diff
@@ -1,6 +1,6 @@
-import Module from './module';
+import IgntModule from './module';
```

Comment:
ts-client is generated right?  Is this just a fork of their repo?

Reply:
Generated but as the ignite issues still persists this has been changed manually